migration/migrator/migrations/course/20231022152303_renaming_late_day_exceptions.py

```diff
@@ -0,0 +1,39 @@
+"""Migration for a given Submitty course database."""
+
+
+def up(config, database, semester, course):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+
+    sql = """ALTER TABLE late_day_exceptions RENAME TO excused_absence_extensions;
```

Comment:
It would be a good idea to check if late_day_exceptions exists before running this. I was able to make it so that my table was named "excused_absence_extensions" before executing this, and it threw an error.